dbos/workflow.go

```diff
@@ -243,17 +243,19 @@ func (h *workflowHandle[R]) processOutcome(outcome workflowOutcome[R], startTime
 		if encErr != nil {
 			return *new(R), newWorkflowExecutionError(workflowState.workflowID, fmt.Errorf("serializing child workflow result: %w", encErr))
 		}
-		recordGetResultInput := recordChildGetResultDBInput{
-			parentWorkflowID: workflowState.workflowID,
-			childWorkflowID:  h.workflowID,
-			stepID:           workflowState.nextStepID(),
-			output:           encodedOutput,
-			err:              outcome.err,
-			startedAt:        startTime,
-			completedAt:      completedTime,
+		recordGetResultInput := recordOperationResultDBInput{
+			workflowID:      workflowState.workflowID,
+			childWorkflowID: h.workflowID,
+			stepID:          workflowState.nextStepID(),
+			output:          encodedOutput,
+			err:             outcome.err,
+			startedAt:       startTime,
+			completedAt:     completedTime,
+			isGetResult:     true,
+			stepName:        "DBOS.getResult",
```

Comment:
We should be able to use the stepName to determine whether this is a get result step (no need for `isGetResult`)